sdk/python/testvectors/nem/namespace_registration.py

```diff
@@ -0,0 +1,39 @@
+from symbolchain import nc
+
+
+def generate(factory):
+	descriptors = {
+		'NamespaceRegistrationTransaction_1': {
+			'signature': nc.Signature(
+				'7BB9FC2F39D56511D8CA4CDA6FDC555481C5C75B0A00BC6198260B607B4F5AEA'
+				'185413F98183646DE6ED208F22DB4AAF6CF0F996E4A7710B33CA873C6DEC180A'),
+			'type': 'namespace_registration_transaction',
+			'version': 0x1,
+			'network': 'testnet',
+			'timestamp': 0x0001E240,
+			'signer_public_key': '8B13C37938D786D97CC374CB8D35279290210684AD3E9D664FD5F40B7096F2C8',
+			'fee': 0x00000000066FF300,
+			'rental_fee_sink': 'TCTXZSZTZGQBWS7MRXTWMOR72LPX2YEVKIXGDIHL',
+			'rental_fee': 0x0000000028697580,
+			# TODO: this is probably wrong in generator, explicit none is required, cause by default it's initialized to empty bytes array
```

Comment:
hm?

Reply:
I think it's a minor bug in generator, which should initialize that `parent_name` to None not to bytes()
(because it is used in conditional)